app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -112,15 +112,69 @@ def raw_destroy
     unsupported_reason_add(:resize, _('The VM is not powered off')) unless current_state == "off"
```

Comment:
We should probably add `unsupported_reason_add(:resize, _('The VM is not connected to a provider')) unless ext_management_system` so that you don't have to worry about the `self.ext_management_system` being `nil` below (which I assume is why you had `self.ext_management_system&.flavors&.each`)